src/argowrapper/workflows/argo_workflows/gwas.py

```diff
@@ -170,12 +170,14 @@ def interpret_gwas_workflow_error(step_name: str, step_log: str) -> str:
         """A static method to interpret the error message in the main-log file 
         of Failed Retry node
         """
-        if step_name=="run-plots" and "mutate" in step_log:
+        if step_name=="run-null-model" and "solve.default" in step_log:
+            show_error = "Small cohort size or unbalanced cohort sizes."
```

Comment:
perhaps we can mention the threshold of at least 700 for a cohort size for continuous outcomes? a message like `Note: for continuous workflows a cohort size of 700 or below may fail` ?